setup.py

```diff
@@ -26,7 +26,8 @@
       ],
       packages=find_packages(exclude=('docs', '*.tests')),
       install_requires=[
-          'ruamel.yaml>=0.15.45', 'scipy', 'matplotlib', 'utm', 'pymap3d'],
+          'ruamel.yaml>=0.15.45', 'scipy', 'matplotlib', 'utm', 'pymap3d',
+          'ffmpeg-python', 'moviepy'],
```

Comment:
I think we should make these optional dependencies, adding them to as a new category in `extra_required` below. 

Need to think about how we handle this on import when not installed. Probably just wrap import with `try`/`except ImportError` which prints out dependencies required.

Reply:
Done 👍